packages/video_player/video_player_android/android/src/test/java/io/flutter/plugins/videoplayer/VideoPlayerTest.java

```diff
@@ -41,6 +47,76 @@ public void before() {
     fakeSurfaceTextureEntry = mock(TextureRegistry.SurfaceTextureEntry.class);
     fakeVideoPlayerOptions = mock(VideoPlayerOptions.class);
     fakeEventSink = mock(QueuingEventSink.class);
+    httpDataSourceFactorySpy = spy(new DefaultHttpDataSource.Factory());
+  }
+
+  @Test
+  public void videoPlayer_BuildsHttpDataSourceFactoryProperlyWhenHttpHeadersNull() {
```

Comment:
Nit: Here and below the B should be lower case, per https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names

Reply:
Also my fault -- apologies!